src/components/WalletUnlock/Ledger.tsx

```diff
@@ -106,7 +106,7 @@ const LedgerDecrypt = ({ formData, onUnlock }: OwnProps) => {
     return <UnsupportedNetwork walletType={translateRaw('x_Ledger')} network={network} />;
   }
 
-  if (!process.env.BUILD_ELECTRON && window.location.protocol !== 'https:') {
+  if (!process.env.IS_ELECTRON && window.location.protocol !== 'https:') {
```

Comment:
Either if (!IS_ELECTRON ...) or `if (!process.env.TARGET_ENV==='electron').

And reading this:
if (!process.env.IS_ELECTRON && window.location.protocol !== 'https:') It seems to mean: "when we're in the browser on not in production"?
Since we use - or should use - https on dev, staging, and prod I don't see when this applies.

Reply:
same